src/Selenium2Library/locators/elementfinder.py

```diff
@@ -4,6 +4,25 @@
 from Selenium2Library.utils import escape_xpath_value, events
 
 
+class ParserLocator(object):
+
+    @classmethod
+    def parse_locator(cls, locator):
+        prefix = None
+        criteria = locator
+        if locator.startswith('//') or locator.startswith('(//'):
```

Comment:
`startswith()` accept a tuple of possible matches like `locator.startswith(('//', '(//'))`.